src/interfaces/ISablierFactoryMerkleBase.sol

```diff
@@ -13,7 +13,7 @@ interface ISablierFactoryMerkleBase is IAdminable {
     //////////////////////////////////////////////////////////////////////////*/
 
     /// @notice Emitted when the accrued fees are collected.
-    event CollectFees(address indexed admin, ISablierMerkleBase indexed campaign, uint256 feeAmount);
+    event CollectFees(address indexed feeRecipient, ISablierMerkleBase indexed campaign, uint256 feeAmount);
```

Comment:
Fee recipient should be 2nd param

Reply:
Caller, campaign, feeRecipient, feeAmount.

These values in this order it should emit right?

Reply:
```solidity
event CollectFees(
        address indexed admin, ISablierMerkleBase indexed campaign, address indexed feeRecipient, uint256 feeAmount
    );
```